src/util/vscodePrompts.ts

```diff
@@ -146,14 +152,21 @@ export async function askForTargetFolder(artifactId: string): Promise<Uri | unde
 
   if (targetFolder && (await exists(path.join(targetFolder.fsPath, artifactId)))) {
     const selection = await askConfirmation(
-      `Folder ${artifactId} already exists inside the ${targetFolder.fsPath} folder. Contents of the ${artifactId} folder and the generated MicroProfile Starter Project will be merged. Are you sure you want to generate into this folder?`
+      `Folder ${artifactId} already exists inside the ${targetFolder.fsPath} folder. Contents of the ${artifactId} folder and the generated MicroProfile Starter Project will be deleted. Are you sure you want to generate into this folder?`
     );
     if (selection === CONFIRM_OPTIONS.YES) {
-      return targetFolder;
+      // delete the existing folder.
+      try {
+        deleteFolder(path.join(targetFolder.fsPath, artifactId));
+      } catch (e) {
+        //the folder is in use
+        vscode.window.showErrorMessage(
+          `Failed to delete folder ${targetFolder.fsPath} because it is being used by another process.`
```

Comment:
This error message should read:
```
Failed to delete folder ${targetFolder.fsPath} because it is being used by another process.  Failed to generate a MicroProfile Starter project.
```